src/Kralizek.Extensions.Configuration.AWSSecretsManager/Internal/SecretsManagerConfigurationProvider.cs

```diff
@@ -190,16 +191,16 @@ private async Task<HashSet<(string, string)>> FetchConfigurationAsync(Cancellati
 
                     var secretValue = await Client.GetSecretValueAsync(new GetSecretValueRequest { SecretId = secret.ARN }, cancellationToken).ConfigureAwait(false);
 
+                    var secretName = secret.Name ?? secretValue.Name;
```

Comment:
I'm not hundred percent sure about this line.
I can replace it with `var secretName = secretValue.Name;`
@Kralizek What do you think?